### PR TITLE
Hack School Introduction & Logistics Page

### PR DESCRIPTION
**Pull request by nik-dange**
Introductory pages (still need to fill it with specific links closer to the start of the school year).

### CODE REVIEW

File: pages/hack/other/test.mdx [hunk 1]
```diff
@@ -0,0 +1 @@
+hello
```
Comment (alexzhang1618) [inline pages/hack/other/test.mdx:1]: is this leftover? perhaps remove

**Review by alexzhang1618 — APPROVED** [review]

**Review by alexzhang1618 — APPROVED** [review]
Nice! For consistency, I think the folder `Hack-School` should be lowercase `hack-school`

**Review by alexzhang1618 — COMMENTED** [review]
I added some comments on Vercel:
1. we can remove the Another page
2. You can configure in meta.json so the sidebar appears as something other than Hack-School

**Review by khushijpatel — APPROVED** [review]
lgtm!

**Pull request merged**